ocs_ci/ocs/cluster.py

```diff
@@ -730,6 +746,142 @@ def time_taken_to_complete_rebalance(self, timeout=600):
+        Setting up the PG / PGP / PG_MIN number of a pool
+        if the pg_num_min is not setting to the pg_num number, the autoscale will
+        set automaticlly the pg_num to 32 (incase you try to set pg_num > 32)
+
+        Args:
+            poolname (str): the pool name that need to be modify
+            pgs (int): new number of PG's
+
+        """
+        for key in ["pg_num", "pgp_num", "pg_num_min"]:
+            cmd = f"ceph osd pool set {poolname} {key} {pgs}"
+            try:
+                logger.debug(f"Try to set {key} to {pgs}")
+                _ = self.toolbox.exec_ceph_cmd(ceph_cmd=cmd, format=None)
+            except Exception as ex:
+                logger.error(f"Failed to setup {key} : {ex}")
+
+    def set_target_ratio(self, poolname, ratio):
+        """
+        Setting the target_size_ratio of a ceph pool
+
+        Args:
+            poolname (str): the pool name
+            ratio (float): the new ratio to set
+
+        """
+        cmd = f"ceph osd pool set {poolname} target_size_ratio {ratio}"
+        try:
+            logger.debug(f"Try to set target_size_ratio on {poolname} to : {ratio}")
+            _ = self.toolbox.exec_ceph_cmd(ceph_cmd=cmd, format=None)
+        except Exception as ex:
+            logger.error(f"Failed to change the ratio : {ex}")
+
+    def get_cephfilesystem_status(self, fsname=None):
+        """
+        Getting the ceph filesystem status
+
+        Args:
+            fsname (str): The filesystem name
+
+        Returnes:
+            bool : true if the filesystem status is `Ready`, false otherwise
+        """
+        res = self.CEPHFS.get(resource_name=fsname)
+        return res.get("status").get("phase") == constants.STATUS_READY
+
+    def create_new_filesystem(self, fs_name):
+        """
+        Creating new filesystem to use in the tests insted of the default one.
+        the new filesystem is identical (parameters wise) to the default filesystem
+
+        Args:
+            fs_name (str):  The name of the filesystem to create
+
+        """
+        # Creating the new filesystem using the default parameters
+        self.cephfs.data["metadata"]["name"] = fs_name
+        self.cephfs.apply(**self.cephfs.data)
+
+        # Verify that the filesystem was created and the cluster if healthy
+        sample = TimeoutSampler(
+            timeout=120, sleep=3, func=self.get_cephfilesystem_status, fsname=fs_name
+        )
+        if not sample.wait_for_func_status(result=True):
+            err_msg = "Can not create new filesystem"
+            logger.error(err_msg)
+            raise exceptions.CephHealthException(err_msg)
+
+    def delete_filesystem(self, fs_name):
+        """
+        Delete a ceph filesystem - not the default one - from the cluster
+
+        Args:
+            fs_name (str): the name of the filesystem to delete
+        """
+        # Make sure the the default filesystem is not deleted.
+        if fs_name == "ocs-storagecluster-cephfilesystem":
```

Comment:
Regardless of pass/fail of "if" condition, line num 806, 807 will be executed. This could be changed accordingly.

Reply:
Good catch @keemano , i replaced it with return